ci/jenkins/test.sh

```diff
@@ -495,13 +499,13 @@ function run_conformance_windows {
     export PATH=$GOROOT/bin:$PATH
     export KUBECONFIG=$KUBECONFIG_PATH
 
-    if [[ "$TESTCASE" == "windows-conformance" ]]; then
+    if [[ "$TESTCASE" =~ "windows-networkpolicy-process" ]]; then
```

Comment:
I notice some code use "==" while some use "=~". We don't need regular expression, right?

Reply:
We need it.
e.g.
https://github.com/antrea-io/antrea/blob/3bbb1068140a7a2420e4703a590671679b96bde7/ci/jenkins/test.sh#L576
We use `=~` here because windows testcases (windows-e2e, windows-conformance, windows-networpolicy) have `windows` in variable `${TESTCASE}`

Reply:
I mean even when it matchs "windows-networkpolicy-process", it sometimes uses "==" and sometimes uses "=~". I think the script should be accurate to express its intention, like what you explained here. We should use "==" when we want exact match.

Reply:
Used `==` for all `"windows-networkpolicy-process"`.